### PR TITLE
Homepage top styles

### LINKED ISSUE #55: Homepage: Top section
Screen Shot 2020-01-23 at 6 08 50 PM

- Header image will be set separately, not based on a featured article. Use the one from the design or the one from their current site for now, a final one is forthcoming.
- Text should be a text field that can be entered, rather than automatically pulling from an internal page
- Option for "more" link should exist under the text section as well
- Quick links will be links to internal pages

### PR DESCRIPTION
**Pull request by benlk**
## Changes

This PR styles the homepage top:

- font scales between 28px and 40px 
- blue background is transparent at desktop sizes and overlaid on the featured image
- on mobile sizes, the blue area drops below the featured image and loses its transparency
- on mobile sizes, the Quick Links menu drops below the featured story's excerpt
- removes top term from template

Screen Shot 2020-01-24 at 22 06 56
Screen Shot 2020-01-24 at 22 07 58

## Why

Resolves #55 

## Testing/Questions

Features that this PR affects:

- Homepage

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [x] Do we need a fallback image if the homepage top story lacks a featured image?

Steps to test this PR:

1. Switch to the IPBS Homepage template in Theme Options > Layout
2. In Appearance > Menus, make sure the Homepage Quick Links menu is populated. If no such menu exists, go to Menu Locations and make one.
3. Make sure the most-recent post on the site has a featured image
4. View the homepage. resizing

### CODE REVIEW

File: wp-content/themes/ipbs-largo/homepages/assets/less/homepage.less [hunk 1]
```diff
@@ -1 +1,112 @@
...
+
+		display: flex;
+		flex-wrap: wrap;
+
+		article {
+			border-bottom: none;
+			flex: 1 1 400px;
+			padding-right: 1 * @measure;
+		}
+		.menu-container {
```
Comment (joshdarby) [inline wp-content/themes/ipbs-largo/homepages/assets/less/homepage.less:29]: Should we add `padding-top: 0;` to this? Currently it's inheriting `padding-top: 3rem;` which is putting the menu out of line with the post title

3rem:
Screen Shot 2020-01-27 at 9 53 13 AM

0:
Screen Shot 2020-01-27 at 9 53 36 AM

**Review by joshdarby — COMMENTED** [review]
> Do we need a fallback image if the homepage top story lacks a featured image?

I would think so, since we can't always guarantee an image exists. If we don't use a fallback image, we should at least make sure the container looks ok. Here's how I currently see it with no image (not sure if this is correct or not though)

Screen Shot 2020-01-27 at 9 55 26 AM

File: wp-content/themes/ipbs-largo/homepages/assets/less/homepage.less (hunk 1 above)
Comment (benlk) [reply wp-content/themes/ipbs-largo/homepages/assets/less/homepage.less:29]: Fixed in 2cb533d:

Desktop:
Screen Shot 2020-01-27 at 15 31 47

Mobile:
Screen Shot 2020-01-27 at 15 31 55

**Comment by benlk** [comment]
6048cc7 gates the positional and transparency styles for `.inner` on a class `.has-featured-image` which is only output by `post_class()` if the post has a featured image. When that class is not present, `.inner` loses its outside padding, and gains a fully-opaque background, while keeping the layout.

Screen Shot 2020-01-27 at 15 48 32
Screen Shot 2020-01-27 at 15 48 42

**Review by joshdarby — APPROVED** [review]

**Pull request merged**